Dockerfile

```diff
@@ -9,6 +9,7 @@ ADD nakadi /nakadi
 RUN find ./nakadi | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
 
 ADD run.sh /nakadi/run.sh
+ADD scm-source.json /scm-source.json
```

Comment:
My only problem is that eventstore is opensource project and the build script is in our private repo.
And we require something that can be done only with help of private repo script. That's why I think that adding scm-source.json to container should be optional.